docs/guide-build.md

```diff
@@ -77,13 +79,13 @@ make k8s-check
 
 which will try pinging from NSCs to NSEs.
 
-You can remove the effects of k8s-deploy with:
+You can remove the effects of helm-install-% with:
 
 ```bash
-make k8s-delete
+make helm-delete-%
 ```
 
-As in the case with `save` and `build`, you can always do this for a particular component, like ```make k8s-nsc-deploy``` or ```make k8s-nsc-delete```.
+As in the case with `save` and `build`, you can always do this for a particular component, like ```make helm-install-nsc``` or ```make helm-delete-nsc```.
```

Comment:
There is no `helm-install-nsc`

Reply:
Essentially we are loosing this target and this should be completely fine.

Reply:
thanks for review. Fixed